goldstone/core/fixtures/core_initial_data.yaml

```diff
@@ -470,4 +470,22 @@
     fields:
         name: service status UP
         description: 'one or more services have changed status to UP'
-        search: de730aab-e0f9-40c7-81c0-4289ddb67e96
\ No newline at end of file
+        search: de730aab-e0f9-40c7-81c0-4289ddb67e96
+
+  - model: core.SavedSearch
+    pk: 21f5c6db-5a2e-41d4-9462-c3cdc03a837b
+    fields:
+      name: nova spawns
+      description: 'Nova spawns.'
+      owner: core
+      index_prefix: goldstone-*
+      doc_type: nova_spawns
+      timestamp_field: '@timestamp'
+      protected: False
+      query: '{
+    "query": {
+       "term": {
+          "event": "finish"
+       }
+    }
+  }'
```

Comment:
No newline at end of file.  This seems to be a new search that is outside the scope of this ticket.  Was there some issue with the spawns data?  If this is a fixture for testing, please create it in the test code during setup, rather than creating a test fixture in production code.